homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
On my system, calling update here prevents the intermittent login issues. Without it, several restarts are required to properly have both USPS sensors appearing. There may be other fixes, this change was based on the older version of the component where the update was being called here but there could be better solutions.